tests/tests_zone.yml

```diff
@@ -86,6 +87,7 @@
               permanent: yes
               masquerade: yes
               state: enabled
+            - set_default_zone: dmz
```

Comment:
In addition - I suggest adding a test like this:
```
firewall:
  - set_default_zone: somezone
  - service: http
```
the `service` without a zone should be added to the default zone, which should be the one set by the prior item